notification/src/models/transformationEvent.ts

```diff
@@ -0,0 +1,32 @@
+import { TransformationEventInterface } from "../interfaces/transformationResults/transformationEventInterface"
+import JobResult from "../interfaces/transformationResults/jobResult"
+
+
+/**
+ * Event send by the Transformation service upon transformation finish
+ */
+export class TransformationEvent implements TransformationEventInterface {
+    pipelineID: number      // ID of the pipeline
+    pipelineName: string    // Name of the pipeline
+
+    dataLocation: string    // url (location) of the pipeline
+
+    jobResult: JobResult            // result of the transformation
+
+    constructor(pipelineID: number, pipelineName: string, jobResult: JobResult, dataLocation: string) {
+        this.pipelineID = pipelineID
+        this.pipelineName = pipelineName
+        this.dataLocation = dataLocation
+        this.jobResult = jobResult
+    }
+
+    /**
+     * Checks if this event is a valid Transformation event,
+     * by checking if all field variables exist and are set.
+     *
+     * @returns     true, if param event is a TransformationEvent, else false
+     */
+    public isValidTransformationEvent(): boolean {
```

Comment:
Same here. I would rather use an interface instead of a class and add a non-class related method (you can have method outside of classes in Typescript) that validates an event:
`export isValidTransformationEvent(event: TransformationEvent): boolean {...}`